src/Utils/Colors.ts

```diff
@@ -6,41 +6,18 @@ export const PrimaryColor = {
 
 export const SecondaryColor = {
   actionblue: '#017EB7',
+  darkblue: '#132647',
   green: '#93BD49',
   darkgreen: '#008464',
   magenta: '#FF00E5',
   violet: '#9013FE',
   orange: '#F5A623',
-
-  // deprecated
-  ivory: '#FFFAEA',
```

Comment:
@westwood846 , did you check that these colours are not being used in the projects?

Reply:
Yes, Employers doesn't use any but there were a lot on DST. I've drafted an MR there already: https://gitlab.glints.com/glints/glints-dst/-/merge_requests/2323

Reply:
Great, thanks!